example/android/app/build.gradle

```diff
@@ -135,13 +158,30 @@ android {
             }
         }
     }
+
+    packagingOptions {
+        pickFirst '**/armeabi-v7a/libc++_shared.so'
+        pickFirst '**/x86/libc++_shared.so'
+        pickFirst '**/arm64-v8a/libc++_shared.so'
+        pickFirst '**/x86_64/libc++_shared.so'
+        pickFirst '**/x86/libjsc.so'
+        pickFirst '**/armeabi-v7a/libjsc.so'
+    }
 }
 
 dependencies {
-    implementation project(':react-native-share')
     implementation fileTree(dir: "libs", include: ["*.jar"])
-    implementation "com.android.support:appcompat-v7:${rootProject.ext.supportLibVersion}"
+    implementation project(':react-native-share')
```

Comment:
Remove this line. RN 0.60+ uses autolinking for native dependencies.

Reply:
No. Separate PR #551

Reply:
#551 is an issue. Did you mean to link a different PR?

Reply:
correction - dependent auto-linking PR is #570

Reply:
#570 - we may have been ships passing in the night with the comment timing there - sorry for the confusion